packages/core/src/tools/error.ts

```diff
@@ -65,3 +74,26 @@ export function toStackTraceString(stack: StackTrace) {
 export function formatErrorMessage(stack: StackTrace) {
   return `${stack.name || 'Error'}: ${stack.message!}`
 }
+
+/** Has to be called at the utmost position of the RUM call stack */
+export function createHandlingStackTrace(): StackTrace {
+  const instrumentationCallDepth = 2
```

Comment:
why 2?

Reply:
2 is number of calls we need to skip in order to reach the user stacktrace.
Here is the two scenarios to report the handling stack
```
addError  -> createHandlingStackTrace
```
or
```
console.log  -> createHandlingStackTrace
```

Reply:
if we stick with this strategy (cf https://github.com/DataDog/browser-sdk/pull/889#discussion_r651768404), I'd be in favor of:
- clarify the strategy in the function comment
- rename this variable in something like `internalFrameToSkip`